app/build.gradle.kts

```diff
@@ -153,6 +153,11 @@ dependencies {
     // scheduling background jobs
     implementation("androidx.work:work-runtime:2.9.0")
 
+    // HTTP Client
+    implementation("io.ktor:ktor-client-core:2.3.8")
+    implementation("io.ktor:ktor-client-cio:2.3.8")
```

Comment:
Other [engines](https://ktor.io/docs/http-client-engines.html#requirements) exist. CIO is the "default" that Ktor talk about, but I see that OkHttp is already in use by `MapFragment.kt`.

Reply:
The OkHttp code in MapFragment is specific to tangram-es which is due to be removed, so don't bother with it.